src/almanack/code_tracker.py

```diff
@@ -0,0 +1,31 @@
+"""
+This module calculates the absolute value of lines of code (LoC) changed (added or removed)
+between the source and target commit hash.
+"""
+
+import pathlib
+
+from .git_parser import get_commit_logs
+
+
+def calculate_loc_changes(repo_path: pathlib.Path, source: str, target: str) -> int:
+    """
+    Finds the total number of code lines changed between the source and target commits.
```

Comment:
Would this be able to parse individual file-level lines of code differences between the source and target commits? If not, consider expanding the function to include this capability (perhaps with an additional parameter which identifies a file within a tree) to address this.